ConsoleAppHomework03/ConsoleAppHomework03/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿try
+{
+    Console.WriteLine("Insert first integer:");
+    int x = int.Parse(Console.ReadLine());
```

Comment:
we've discussed to ask the user one more time if he enters not a number

Reply:
fixed